btax/front_end_util.py

```diff
@@ -0,0 +1,260 @@
+from __future__ import unicode_literals
+from collections import defaultdict
+import os
+from btax.parameters import DEFAULT_ASSET_COLS, DEFAULT_INDUSTRY_COLS
+
+# Row labels, in order, including minor headings like "Durable goods"
+BTAX_TABLE_ASSET_ORDER = ("Equipment", "Mainframes", "PCs", "DASDs", "Printers", "Terminals", "Tape drives", "Storage devices", "System integrators", "Communications", "Nonelectro medical instruments", "Electro medical instruments", "Nonmedical instruments", "Photocopy and related equipment", "Office and accounting equipment", "Nuclear fuel", "Other fabricated metals", "Steam engines", "Internal combustion engines", "Metalworking machinery", "Special industrial machinery", "General industrial equipment", "Electric transmission and distribution", "Light trucks (including utility vehicles)", "Other trucks, buses and truck trailers", "Autos", "Aircraft", "Ships and boats", "Railroad equipment", "Household furniture", "Other furniture", "Other agricultural machinery", "Farm tractors", "Other construction machinery", "Construction tractors", "Mining and oilfield machinery", "Service industry machinery", "Household appliances", "Other electrical", "Other", "Structures", "Office", "Hospitals", "Special care", "Medical buildings", "Multimerchandise shopping", "Food and beverage establishments", "Warehouses", "Mobile structures", "Other commercial", "Manufacturing", "Electric", "Wind and solar", "Gas", "Petroleum pipelines", "Communication", "Petroleum and natural gas", "Mining", "Educational and vocational", "Lodging", "Amusement and recreation", "Air transportation", "Other transportation", "Other railroad", "Track replacement", "Local transit structures", "Other land transportation", "Farm", "Water supply", "Sewage and waste disposal", "Public safety", "Highway and conservation and development", "Inventories", "Land")
```

Comment:
@PeterDSteinberg, One of the outstanding issues is to have an "All Investments" line.  Such a category is already in the B-Tax tables.  Shall we add to this list to take care of that issue as well?